webrtc.html

```diff
@@ -11467,21 +11468,26 @@ <h2>
                     </p>
                   </li>
                   <li>
+                    <div id="codec-match-without-level-id">
                     <p>
                       If either (but not both) of <var>first</var>.{{RTCRtpCodec/sdpFmtpLine}}
                       and <var>second</var>.{{RTCRtpCodec/sdpFmtpLine}} are [= map/exist | missing =],
                       or if they both [=map/exist=] and <var>first</var>.{{RTCRtpCodec/sdpFmtpLine}}
                       is different from <var>second</var>.{{RTCRtpCodec/sdpFmtpLine}}, return
-                      <code>false</code>.
+                      <code>false</code>. Two FMTP lines are the same as long as they convey the same
+                      information, including omitting parameters with well defined defaults or using
+                      different order. In this step, the user agent MUST consider two FMTP lines
+                      identical if the "media format" are the same but the highest complying bitstream
```

Comment:
```suggestion
                      identical if the "media format" is the same but the highest complying bitstream
```

Reply:
Done + reworded

Reply:
The text is ambiguous because not all fmtp parameters are included in the "media format" for matching purposes. As an example, the statement applies to H.265 but not H.264 where `profile-level-id` combines `profile` with `level-id`.